README.md

```diff
@@ -1 +1,16 @@
-# cc-monitor
+# CalCentral Monitor
+
+## Build
+Run the NodeJS server with:
+
+```bash
+npm start
+```
+And run the webpack (front end) server with:
+
+```bash
+webpack-dev-server
+```
+The web application can be accessed at [localhost:8080](http://localhost:8080/).
+## Screenshot
+![CalCentral Monitor Page Screenshot](http://s28.postimg.org/4u4govh7h/Screen_Shot_2016_04_07_at_4_58_21_PM.png)
```

Comment:
@winstonkim could we also add some comments about how `.env_template` and `.env` work?